packages/lib/onedrive-api.js

```diff
@@ -179,32 +192,34 @@ class OneDriveApi {
 					let contentLength = null;
 					if (i === numberOfChunks - 1) {
 						// Last fragment. It is not ensured that the last fragment is a multiple of 327,680 bytes as recommanded in the api doc. The reasons is that the docs are out of day for this purpose: https://github.com/OneDrive/onedrive-api-docs/issues/1200#issuecomment-597281253
-						endByte = fileSize - 1;
-						contentLength = fileSize - ((numberOfChunks - 1) * chunkSize);
+						endByte = byteSize - 1;
+						contentLength = byteSize - ((numberOfChunks - 1) * chunkSize);
 					} else {
 						endByte = (i + 1) * chunkSize - 1;
 						contentLength = chunkSize;
 					}
-					this.logger().debug(`${options.path}: Uploading File Fragment ${(startByte / 1048576).toFixed(2)} - ${(endByte / 1048576).toFixed(2)} from ${(fileSize / 1048576).toFixed(2)} Mbit ...`);
+					this.logger().debug(`Uploading File Fragment ${(startByte / 1048576).toFixed(2)} - ${(endByte / 1048576).toFixed(2)} from ${(byteSize / 1048576).toFixed(2)} Mbit ...`);
```

Comment:
I removed the options.path because this is undefined in case of note and it was not really useful anyway.